src/sos_matlab/converter.py

```diff
@@ -33,13 +33,10 @@ def convert(self, script_file, notebook_file, args=None, unknown_args=None):
             metainfo = {}
             if args.use_sos:
                 metainfo = {'kernel': 'SoS'}
-            cells = [
-                new_code_cell(
-                    # remove any trailing blank lines...
-                    source=src.read(),
-                    execution_count=1,
-                    metadata=metainfo)
-            ]
+            
+            lines = re.split(r'\n[^\S\t]+', src.read())
```

Comment:
<img width="1334" alt="image" src="https://user-images.githubusercontent.com/9889312/181595408-70dd5866-373c-4292-b66a-f6fb205dba9b.png">

Reply:
Why not `r'\n\s*\n'`?

Reply:
That is cleaner; I'll change it